internal/garbageCollector/garbageCollector.go

```diff
@@ -0,0 +1,85 @@
+package garbageCollector
```

Comment:
```suggestion
package garbagecollector
```
https://blog.golang.org/package-names